README.md

```diff
@@ -370,7 +370,7 @@ let w = new Signal.subtle.Watcher(() => {
 // itself on the microtask queue whenever one of its dependencies might change
 export function effect(cb) {
     let destructor;
-    let c = new Signal.Computed(() => { destructor?.(); destructor = cb(); });
+    const c = new Signal.Computed(() => { destructor?.(); destructor = cb(); });
```

Comment:
Same should we go multiline here ?

Reply:
Well, I know this is sort of personal taste, but in my opinion this is acceptable. Lambda functions are often one-liners and the content in this case is not too complex. I assume a reader is used to reading the rest of the line. Understanding the code is going to be the issue (why it does what it does?), but not the style.